### ISSUE TITLE
Can't push or get remote repo changes

### ISSUE THREAD

**Issue by maicol07** [WeblateOrg/weblate#3581]
**Describe the bug**
When I press the "Push" or the "Get" button in the Repo mantainance page, action fails and I receive an error email.
Error when getting from remote repo:
```FileParseError at /update/scheduled-exams/webapp/ No such file or directory```
Error when pushing changes:
```FileParseError at /push/scheduled-exams/webapp/ No such file or directory```

**Expected behavior**
Weblate should push or get changes correctly...

**Server configuration and status**
```
 * Weblate: 3.11.2
 * Django: 2.2.5
 * siphashc: 1.2
 * Whoosh: 2.7.4
 * translate-toolkit: 2.4.0
 * lxml: 4.4.1
 * Pillow: 6.2.1
 * bleach: 3.1.0
 * six: 1.14.0
 * python-dateutil: 2.8.1
 * social-auth-core: 3.2.0
 * social-auth-app-django: 3.1.0
 * django-crispy-forms: 1.8.1
 * oauthlib: 3.1.0
 * django-compressor: 2.3
 * djangorestframework: 3.10.3
 * django-appconf: 1.0.3
 * user-agents: 2.0
 * filelock: 3.0.12
 * setuptools: 41.2.0
 * jellyfish: 0.7.2
 * openpyxl: 2.6.3
 * celery: 4.3.0
 * kombu: 4.6.7
 * celery-batches: 0.2
 * translation-finder: 1.7
 * html2text: 2019.9.26
 * pycairo: 1.18.1
 * pygobject: 3.34.0
 * diff-match-patch: 20181111
 * requests: 2.22.0
 * django-redis: 4.10.0
 * hiredis: 1.0.1
 * sentry_sdk: 0.14.1
 * Cython: 0.29.14
 * misaka: 2.1.1
 * GitPython: 3.0.5
 * borgbackup: 1.1.10
 * Python: 3.6.8
 * Git: 2.17.1
 * psycopg2-binary: 2.8.3
 * phply: 1.2.5
 * chardet: 3.0.4
 * ruamel.yaml: 0.16.5
 * zeep: 3.4.0
 * aeidon: 1.6.0
 * hub: 2.17.1
 * Database backends: django.db.backends.postgresql
 * Cache backends: default:RedisCache, avatar:FileBasedCache
 * Email setup: django.core.mail.backends.smtp.EmailBackend: mail.maicol07.it
 * Celery: redis://localhost:6379, redis://localhost:6379, regular
 * Platform: Linux 5.5.5-050505-generic (x86_64)
```

**Comment by nijel** [comment]
Maybe your configuration got out of sync with the repo and the translation files no longer exist there?

**Comment by maicol07** [comment]
> 
> 
> Maybe your configuration got out of sync with the repo and the translation files no longer exist there?

@nijel how can I see this?

**Comment by nijel** [comment]
The files matching filemask are no longer present in the repo?

**Comment by maicol07** [comment]
@nijel I have only Italian and English in this project.
I see the English po and the Italian pot. Should I create an Italian po file?

**Comment by nijel** [comment]
Check the email with error details to see what file is the problem...

**Comment by maicol07** [comment]
Ok, italian translation file was missing. So now I get the duplicate translation error but if I remove the Italian po file (italian is set as source language) I get the previous error (FileNotFoundError). What should I do?

**Comment by nijel** [comment]
How is your component configured?

**Comment by maicol07** [comment]
@nijel here you go:
![immagine](https://user-images.githubusercontent.com/9463142/76679178-70edd000-65de-11ea-981a-8358207ce0c3.png)

**Comment by nijel** [comment]
Can you please post full traceback for the error? You should find in the logs or email (if ADMINS are properly configured).

Besides that, you have `.pot` as a template and `.po` as a file mask, so these two should not influence each other. Removing Italian `.po` file should be okay.

**Comment by maicol07** [comment]
Here you go: https://pastebin.com/XEn5GL0b

If I delete the file, I get the error

**Comment by nijel** [comment]
Do you delete the file directly in Weblate repo?

**Comment by maicol07** [comment]
> 
> 
> Do you delete the file directly in Weblate repo?

@nijel yes. After I deleted it, I push or pull from the repo and the error shows up

**Comment by nijel** [comment]
Okay, that error should be fixed in 7bc88fa, but still it's safer to do such changes outside Weblate.

**Comment by maicol07** [comment]
Ok thank you. I'll try it with next version
